Mage/src/main/java/mage/util/CardUtil.java

```diff
@@ -901,6 +901,16 @@ public static String getBoostText(DynamicValue power, DynamicValue toughness, Du
         return sb.toString();
     }
 
+    public static Outcome getBoostOutcome(DynamicValue power, DynamicValue toughness) {
+        if (toughness.getSign() < 0) {
+            return Outcome.Removal;
+        }
+        if (power.getSign() < 0) {
+            return Outcome.UnboostCreature;
+        }
+        return Outcome.BoostCreature;
+    }
+
```

Comment:
Is it Outcome.Removal so that the AI knows that if they use it repeatedly they can remove a creature with it? (assuming that power and toughness here represent the change in power and toughness, and not the final values).

Reply:
I'm not that familiar with the AI but I used `Outcome.Removal` here because of `isEffectRemoval` in RateCard.java (which is used by bots to make draft picks and to build their sealed decks). The assignment of `Outcome`s in xmage seems to be pretty haphazard in general.